deployment/network-operator/templates/tests/test-rping.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if and .Values.deployCR .Values.devicePlugin.deploy .Values.secondaryNetwork.deploy .Values.secondaryNetwork.multus.deploy .Values.secondaryNetwork.cniPlugins.deploy }}
+apiVersion: mellanox.com/v1alpha1
+kind: MacvlanNetwork
```

Comment:
ATM This is created regardless if tests are run or not right ?

if you add the test hook annotation it will only be created when helm test is invoked however
you probably hit race where the pod was created but the macvlan net-attach-def was not created yet.

also i wonder when is this network deleted ?

Reply:
Changed to be deployed on test with delete policy for failure and success,
tested it for success and worked
for failure it is not removed, maybe issue with helm
https://helm.sh/docs/topics/charts_hooks/